src/allotropy/parsers/agilent_gen5/agilent_gen5_structure.py

```diff
@@ -87,8 +89,16 @@ def create(cls, data: SeriesData, file_name: str) -> HeaderData:
             well_plate_identifier=plate_identifier or data.get(str, "Plate Number"),
             model_number=data.get(str, "Reader Type:"),
             equipment_serial_number=data.get(str, "Reader Serial Number:"),
+            plate_well_count=plate_well_count,
         )
 
+    @staticmethod
+    def _extract_well_count(plate_type: str) -> float:
+        match = re.search(r"\d+", plate_type)
```

Comment:
Let's search for a specific list of possible well counts, to avoid using a random number identifier

```suggestion
        match = re.search(rf"({'|'.join(str(count) for count in constants.POSSIBLE_WELL_COUNTS)})", plate_type).groups()
```

Where `constants.POSSIBLE_WELL_COUNTS = [48, 96, 384, 1536]`

cc: @james-leinas to approve

Reply:
@nathan-stender, here would be my suggested list of possible well counts [1, 2, 4, 6, 8, 12, 24, 48, 72, 96, 384, 1536, 3456]. Gen5/Gen5 Image offers a lot of compatibility of plate types. Agree that this will avoid arbitrary numbers being flagged for plate well count.

Reply:
Sounds good to me